README.md

```diff
@@ -611,3 +613,56 @@ status:
   binding:
     name: kafka-eftg9
 ```
+
+## Role-Based Access Control (RBAC)
+
+Kubernetes clusters often utilize Role-based access control (RBAC) to authorize subjects to perform specific actions on resources. When operating in a cluster with RBAC enabled, the service binding reconciler needs permission to read resources that provisioned a service and write resources that services are projected into. This extension defines a means for third-party CRD authors and cluster operators to expose resources to the service binding reconciler. Cluster operators **MAY** impose additional access controls beyond RBAC.
+
+Cluster operators and CRD authors **MAY** opt-in resources to service binding by defining a ClusterRole with a label matching `service.bindings/controller=true`. For Provisioned Service-able resources the `get`, `list`, and `watch` verbs **SHOULD** be granted. For PodSpec-able resources the `get`, `list`, `watch`, `update`, and `patch` verbs **SHOULD** be granted.
```

Comment:
`verb` requirements should be **MUST** rather than **SHOULD** within the context of optional support for this `ClusterRole` definition.

Reply:
you mean must be **MUST** ;)